stdlib/public/Platform/glibc.modulemap.gyb

```diff
@@ -36,6 +36,13 @@ module SwiftGlibc [system] {
   link "execinfo"
 % end
 
+% if CMAKE_SDK in ["FREEBSD"]:
+  module cdefs {
+    header "${GLIBC_INCLUDE_PATH}/sys/cdefs.h"
```

Comment:
Looking at this, I wonder whether this should be `GLIBC_ARCH_INCLUDE_PATH` as it's a sys/ header. I'm not entirely sure, though.

Reply:
Is the path not `/usr/include/sys/cdefs.h` as libbsd would have me believe?

Reply:
No, you're right, the file is there.